docs/spec/manifest-v2-2.md

```diff
@@ -98,7 +103,9 @@ image manifest based on the Content-Type returned in the HTTP response.
         - **`features`** *array*
 
             The optional features field specifies an array of strings, each
-            listing a required CPU feature (for example `sse4` or `aes`).
+            listing a required CPU feature (for example `sse4` or `aes`) or
```

Comment:
I'd prefer to keep this restricted to CPU features, as that was the original intent.

Would it be out of line to have "os.features"?